### ISSUE TITLE
Contact Page

### ISSUE THREAD

**Issue by teaboz** [doodlesacademy/doodles#23]
Needs a contact form.

**Comment by rjsheperd** [comment]
Which email should this go to?

**Comment by teaboz** [comment]
tempest@doodles-academy.org

On Sat, Apr 2, 2016 at 4:52 PM, Richard Sheperd <notifications@github.com>
wrote:

> Which email should this go to?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/doodlesacademy/doodles/issues/23#issuecomment-204828099>
>



-- 
...
Tempest NeuCollins
*Executive Director*
*...*
*Our mission is to provide the tools necessary for art lessons that inspire
students to meet their creative potential. *
*Instagram <https://instagram.com/doodlesacademy/> & **Facebook
<https://www.facebook.com/DoodlesAcademy> *
*doodles-academy.org <http://doodles-academy.org>*
*tempest@doodles-academy.org <tempest@doodles-academy.org>*

**Comment by phantummm** [comment]
@rjsheperd Did you start on this? I'm not sure how mailers work on Heroku, since I doubt we have access to sendmail.

**Comment by phantummm** [comment]
Details pertinent to this are here: https://github.com/doodlesacademy/doodles/issues/47

TL;DR: the contact form will be built to use the same fields/options as the MailChimp subscription and, in fact, checking the "subscribe" box will really just create a subscription on the backend.
